tests/std/tests/usual_17_matrix.lst

```diff
@@ -24,6 +24,7 @@ PM_CL="/EHsc /MTd /D_ITERATOR_DEBUG_LEVEL=2 /std:c++latest /await"
 PM_CL="/EHsc /MTd /D_ITERATOR_DEBUG_LEVEL=2 /std:c++latest /analyze:only"
 PM_CL="/Za /EHsc /MD /std:c++latest /permissive-"
 PM_CL="/Za /EHsc /MDd /std:c++latest /permissive-"
+PM_CL="/EHsc /MD /std:c++17 /permissive- /Zc:preprocessor"

```

Comment:
Until we maybe come up with a better way of denoting the configuration of a test in the test name I'd like to see additions to these files always be appended so we don't change the configuration numbers of the following configurations.

Reply:
Could we add this to 14 and 16 instead? That way we get both IDL0 and IDL2 coverage and don't add another 'configuration way'. (Ditto others)

Reply:
14 and 16 being line numbers?

Reply:
I think that's what I meant